pipe/pipe.py

```diff
@@ -20,20 +20,23 @@
     'OSSINDEX_PASSWORD': {'type': 'string', 'required': False, 'default': ''},
     'NVD_API_KEY': {'type': 'string', 'required': False, 'default': ''},
     'DISABLE_OSSINDEX': {'type': 'boolean', 'required': False, 'default': False},
+    'UPDATE_DB': {'type': 'boolean', 'required': False, 'default': False},
     'EXTRA_ARGS': {'type': 'string', 'required': False, 'default': ''},
     }
 
 class OWASPDependencyCheck(Pipe):
     owasp_failure   = False;
     disableOssIndex = False;
+    updateDb = False;
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.scan_path              = self.get_variable('SCAN_PATH')
         self.suppression_path       = self.get_variable('SUPPRESSION_FILE_PATH')
         self.ossindexusername       = self.get_variable('OSSINDEX_USERNAME')
         self.ossindexpassword       = self.get_variable('OSSINDEX_PASSWORD')
-        self.nvdapikey             = self.get_variable('NVD_API_KEY')
+        self.nvdapikey              = self.get_variable('NVD_API_KEY')
+        self.updatedb               = self.get_variable('NVD_API_KEY')
```

Comment:
Copypasta, just need to update to UPDATE_DB

Reply:
Ooops, I fixed this but forgot to commit it :sweat_smile: